internal/local/app.go

```diff
@@ -49,6 +49,20 @@ func (a App) Option() string {
 	return a.AppData.Name()
 }
 
+// hasValidConfigVersion returns whether or not the config file and version number match
```

Comment:
[nit] prob doesn't need a comment anymore since the func is un-exported but feel free to leave it

Reply:
Is the placement of the function in this file OK? I'm not sure if we have any conventions around putting private functions towards the bottom of the file or anything like that.